common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -802,3 +785,42 @@ describe 'Problem', ->
 
       # verify that codemirror textarea has correct `aria-describedby` attribute value
       expect($(CodeMirrorTextArea).attr('aria-describedby')).toEqual('cm-editor-exit-message-101 status_101')
+
+
+  describe 'show answer button', ->
+
+    radioButtonProblemHtml = readFixtures('radiobutton_problem.html')
+    checkboxProblemHtml = readFixtures('checkbox_problem.html')
+
+    beforeEach ->
+      @problem = new Problem($('.xblock-student_view'))
+
+      @clickShowAnswer = (ProblemHtml) =>
+        $('#input_example_1').replaceWith(ProblemHtml)
+        $('#input_1_1_1').attr('checked', true)
+        @problem.el.find('.show').trigger('click')
+        @problem.el.find('div.choicegroup label:nth-child(2)').addClass('choicegroup_correct')
+        @problem.el.find('div.choicegroup .choicegroup_correct').append('<span class="status correct"></span>')
+        @problem.el.find('.show').attr('disabled', 'disabled')
+
+      @submitCheckboxOrRadio = =>
+        # verify that by changing the selected answer, 'show answer button is no longer disabled'
```

Comment:
Selected answer is not getting changed here, comment may confuse reader. I think we don't need to explain the code by writing comments.